css/60_photos.css

```diff
@@ -381,7 +381,9 @@ label.streetside-hires {
 }
 
 .mly-wrapper .mapillary-attribution-container .mapillary-attribution-username {
```

Comment:
It looks like this for me <img width="188" alt="image" src="https://github.com/openstreetmap/iD/assets/111561/512824c8-b4ec-4542-8d42-15f91db83c35">

We need to also set the following, so that the height for the name has the same rules that `.mapillary-attribution-date` uses.

With the height-reset it looks like this <img width="138" alt="image" src="https://github.com/openstreetmap/iD/assets/111561/0f12d334-e5c5-4696-85d9-ff6f6e05092d"> (I did not add the padding in this example, yet)

```suggestion
.mapillary-attribution-compact .mapillary-attribution-image-container {
  height: auto;
}

.mly-wrapper .mapillary-attribution-container .mapillary-attribution-username {
```

Reply:
Thanks, that works much better! updated bf184e21c